extension/data.ts

```diff
@@ -86,6 +86,17 @@ interface DeinflectionRuleGroup {
   rules: DeinflectionRule[];
 }
 
+// prettier-ignore
+const CHAR_CODES_BELOW_0x3000_USED_IN_WORDS_IN_DICTIONARY = [
```

Comment:
Using this reduces the scope of the change but also means the problem could crop up again if a new symbol is added....

As you were working on this did you come across any problems if we just remove the check altogether? It might have been added for performance reasons only...

Reply:
Removing the if-check for character codes under 0x3000 is much more simple and it doesn't appear to introduce any performance problems when I test it locally. Thanks!